### PR TITLE
Updated tutorial text

### PR DESCRIPTION
**Pull request by satyyam11**
This PR updates the Elasticsearch ILM tutorial to make things clearer and easier to follow.
I’ve cleaned up the steps for customizing ILM policies, explained the rollover, warm, and delete phases a bit more, and made sure everything’s easy to understand.
Basically, it's to help people better manage their ILM policies based on their needs, without all the confusion.

### CODE REVIEW

**Comment by satyyam11** [comment]
I signed the document now what

**Comment by satyyam11** [comment]
Shalll I raise a PR for image change as well?

File: docs/reference/ilm/example-index-lifecycle-policy.asciidoc [hunk 1]
```diff
@@ -43,19 +41,18 @@ To complete this tutorial, you'll need:
 * An {es} cluster with hot and warm data tiers.
 
 ** {ess}:
-Elastic Stack deployments on {ess} include a hot tier by default. To add a warm
-tier, edit your deployment and click **Add capacity** for the warm data tier.
-+
+Elastic Stack deployments on {ess} automatically include hot and warm tiers. If needed, you can add additional warm capacity by editing your deployment and selecting **Add capacity** for the warm data tier.
```
Comment (dakrone) [inline docs/reference/ilm/example-index-lifecycle-policy.asciidoc:44]: They do not automatically include a `warm` tier. They include a `hot` and `content` tier.
```suggestion
Elastic Stack deployments on {ess} automatically include hot and content tiers. If needed, you can add additional warm capacity by editing your deployment and selecting **Add capacity** for the warm data tier.
```

File: docs/reference/ilm/example-index-lifecycle-policy.asciidoc [hunk 2]
```diff
@@ -43,19 +41,18 @@ To complete this tutorial, you'll need:
...
++ 
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-ess-add-warm-data-tier.png[Add a warm data tier to your deployment]
+image::images/ilm/tutorial-ilm-add-warm-data-tier.png[Add a warm data tier to your deployment]
```
Comment (dakrone) [inline docs/reference/ilm/example-index-lifecycle-policy.asciidoc:47]: Is there a reason you changed this image link? As part of this the pngs should be updated to show the newer UI.

File: docs/reference/ilm/example-index-lifecycle-policy.asciidoc [hunk 3]
```diff
@@ -84,14 +81,13 @@ To save the `logs@lifecycle` policy as a new policy in {kib}:
 . On the **Edit policy logs** page, toggle **Save as new policy**, and then provide a new name for the policy, for example, `logs-custom`.
 
 The `logs@lifecycle` policy uses the recommended rollover defaults: Start writing to a new
-index when the current write index reaches 50GB or becomes 30 days old.
+index when the primary shard reaches 50GB or the index becomes 30 days old.
 
 To view or change the rollover settings, click **Advanced settings** for the hot
-phase. Then disable **Use recommended defaults** to display the rollover
-settings.
+phase. Adjust the **Minimum primary shard size** and other rollover settings as needed.
```
Comment (dakrone) [inline docs/reference/ilm/example-index-lifecycle-policy.asciidoc:87]: A user will still need to uncheck the "Use recommended defaults" right?

File: docs/reference/ilm/example-index-lifecycle-policy.asciidoc [hunk 4]
```diff
@@ -84,14 +81,13 @@ To save the `logs@lifecycle` policy as a new policy in {kib}:
...
 
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]
+image::images/ilm/tutorial-ilm-hotphaserollover-new.png[View rollover defaults]
```
Comment (dakrone) [inline docs/reference/ilm/example-index-lifecycle-policy.asciidoc:90]: This image doesn't exist, do you need to add it to git so it's committed for this PR?

File: docs/reference/ilm/example-index-lifecycle-policy.asciidoc [hunk 5]
```diff
@@ -102,8 +98,8 @@ indices. You can modify your copy of the policy to meet your performance require
 manage resource usage.
 
 . Activate the warm phase and click **Advanced settings**.
-+
---
++ 
+--
```
Comment (dakrone) [inline docs/reference/ilm/example-index-lifecycle-policy.asciidoc:102]: There is a lot of unnecessary whitespace changes in this file, can you undo those?

**Review by dakrone — CHANGES_REQUESTED** [review]
Thanks for working on this @satyyam11, but we need to update the images as part of this PR in addition to changing the text. Could you make those image updates as well?

**Comment by satyyam11** [comment]
Any updates?

**Pull request closed without merging**